routes/ninja-routes.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+const Ninja = require('../models/ninjas')
+
+module.exports = function(router) {
+  router.get('/api/ninja', (req, res) => {
```

Comment:
So now that we are using mongo, we will be passing an id into your get endpoint. '/api/ninja/:id'.  Mostly, because thats how you'll look up an item in mongo